pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -93,6 +95,10 @@ in
   # This can make it slightly faster to load this library into GHCi, but takes
   # extra disk space and compile time.
   enableLibraryForGhci ? false
+  # Set this to a previous build of this same package to reuse the intermediate
+  # build products from that prior build as a starting point for accelerating
+  # this build
+, previousIntermediates ? null
```

Comment:
New arguments should also be documented in the nixpkgs manual.

Reply:
Done.